src/main/java/db/TransactionReader.java

```diff
@@ -0,0 +1,8 @@
+package db;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface TransactionReader {
+    List<FruitTransaction> readTransactionsFromFile(String fileName);
```

Comment:
IO logic is being combined with business logic, move string to transaction mapping logic to the separate TransactionMapper class
```suggestion
public interface FileReader {
    List<String> read(String fileName);
```